russian/vodka/vodka.md

```diff
@@ -1,2 +1,7 @@
 Живительная влага мягко разливается по измученному организму и невероятным
 усилием воли получается наконец открыть глаза.
+
+Порошенько нажрамшись, я подхватил заготовленное ведро бобров и вручил его зубаке. И что вы думаете она сделала?
+[Радостно засмеялась, увидев близких родственников](ahaha/ahaha.md)
+[Перестала вести лекцию, замерев от восторга](zamerla/zamerla.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.